content/en/agent/faq/linux-agent-2022-key-rotation.md

```diff
@@ -116,6 +116,8 @@ Otherwise, the command returns a non-0 exit code and the following output:
 package gpg-pubkey-fd4bf915 is not installed
 ```
 
+Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is rotated.
```

Comment:
To be more precise, should this be "This key file will be updated with the new key as soon as the repository metadata gets signed with the new key."?

Reply:
```suggestion
Alternatively, you can check if you `datadog.repo` file contains https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public as one of the `gpgkey` entries. This key file will be updated with the new key as soon as it is in use.
```
What about "as soon as it is in use"? Since we don't clarify the difference between signing the metadata vs the actual packages anywhere in these docs.

Reply:
Yeah, I think that's good 👍